### PR TITLE
Create desktop version

### PR DESCRIPTION
**Pull request by pirc25**
### Create desktop version

- Added styles to the desktop version.
- Added grids.
- Added responsive web designer.
- Added flex.
- Approve linters test.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -97,11 +97,11 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
...
         </ul>
+        <button type="button" class="see-project-card" id="button-1">
+          See Project
+        </button>
       </div>
-      <button type="button" class="see-project-card">
-        See Project
-      </button>
-      <div class="card-content">
+      <div class="card-content" id="card-content-2">
```
Comment (gtekle) [inline index.html:104]: - [x] The `see project` button should only be display on a project card on mouse hover and shouldn't be visible on the other on desktop version unlike the mobile version.

- [x] The content of the card should be horizontally left and vertically bottom aligned as in the Figma design.
#### Design
project-see-project-button-desk-figma
#### Impl
project-see-project-button-desk-impl

File: index.html [hunk 2]
```diff
@@ -157,12 +158,28 @@ <h3 class="work-title">Profesional Art Printing Data</h3>
...
       </div>
-      <button type="button" class="see-project-card">
-        See Project
-      </button>
     </div>
   </section>
+
   <section class="about-me" id="about-me">
     <div class="resume">
       <h1>About me</h1>
```
Comment (gtekle) [inline index.html]: - [x] You should have only one `<h1>` tag in an html document for semantic reasons.

Comment (gtekle) [inline index.html]: - [x] Kindly add left margin/padding to try make it look like in the Figma design.
#### Design
about-section-left-figma
#### Impl
about-section-left-impl

File: index.html [hunk 3]
```diff
@@ -175,39 +192,45 @@ <h1>About me</h1>
...
+      </div>
+      <div class="frameworks-div">
+        <ul class="competence-list">
+          <li class="list-title frameworks">
+            Frameworks
+          </li>
+        </ul>
+        <ul class="competence-list">
+          <li class="competence-item">Bootstrap</li>
+          <li class="competence-item">React</li>
```
Comment (gtekle) [inline index.html:216]: - [x] The Language, Frameworks and skills list should be left aligned on desktop version while they are center aligned on mobile. Kindly update as in the Figma design.
#### Design
language-and-skills-section-left-figma
#### Imp
language-and-skills-section-left-impl
l

File: index.html [hunk 4]
```diff
@@ -175,39 +192,45 @@ <h1>About me</h1>
...
+        <ul class="competence-list">
+          <li class="competence-item">Github</li>
+          <li class="competence-item">Codepen</li>
+          <li class="competence-item">Codesandbox</li>
+          <li class="competence-item">Replit</li>
+        </ul>
+      </div>
     </div>
   </section>
   <section class="contact">
```
Comment (gtekle) [inline index.html:236]: - [x] Kindly use the proportional width for the paragraph on the left and the form on right hand side in the contact-me section as in the Figma design.
#### Design
contact-me-parag-long-figma
#### Imp
contact-me-parag-long-impl
l

File: index.html [hunk 5]
```diff
@@ -86,7 +86,7 @@ <h2 class="multi-post">Multi-post stories</h2>
       </button>
     </div>
```
Comment (gtekle) [inline index.html:87]: - [ ] Kindly add right and left margin/padding to the header/navbar section as in the Figma design.

- [ ]  Kindly adjust the gap between the social media icons on the left and the headline paragraph as in the Figma design.
#### Design
header-and-headline-desk-figma
#### Impl
header-and-headline-desk-impl

**Review by gtekle — CHANGES_REQUESTED** [review]
Hi @pirc25 :wave: , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
this-close


## Highlights
:heavy_check_mark: Professional Readme
:heavy_check_mark: Good use of Github flow
:heavy_check_mark: Meaningful commit messages

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly check the inline comments below.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 2 above)
Comment (pirc25) [reply index.html]: Thank you.

Comment (pirc25) [reply index.html]: Thanks for the review.

File: index.html (hunk 3 above)
Comment (pirc25) [reply index.html:216]: Thanks.

File: index.html (hunk 4 above)
Comment (pirc25) [reply index.html:236]: Done.

File: index.html (hunk 3 above)
Comment (pirc25) [reply index.html:216]: Done

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
Hi <STUDENT_MENTION>, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._
 
- [ ] Your linters are not working right now. I think this happened with your latest commit. Make sure it's working when you submit the Pull Request. 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

- [ ] [OPTIONAL} Your Pull Request must follow correct gitflow. Right now you are using Github flow. Please check [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/4_desktop_version.md) once again. 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request closed without merging**